activities-sticky-queues/workflow.go

```diff
@@ -0,0 +1,44 @@
+package activities_sticky_queues
+
+import (
+	"path/filepath"
+	"time"
+
+	"github.com/google/uuid"
+	"go.temporal.io/sdk/workflow"
+)
+
+// FileProcessingWorkflow is a workflow that uses stick activity queues to process files
+// on a consistent host.
+func FileProcessingWorkflow(ctx workflow.Context) (err error) {
+	ao := workflow.ActivityOptions{
+		// Note the use of scheduleToCloseTimeout.
+		// The reason this timeout type is used is because this task queue is unique
+		// to a single worker. When that worker goes away, there won't be a way for these
+		// activities to progress.
+		ScheduleToCloseTimeout: time.Minute,
+	}
+	ctx = workflow.WithActivityOptions(ctx, ao)
+	var stickyTaskQueue string
+	err = workflow.ExecuteActivity(ctx, "GetStickyTaskQueue").Get(ctx, &stickyTaskQueue)
```

Comment:
You want this activity not to have scheduleToClose timeout and the rest to have it